superset-frontend/src/explore/components/controls/CollectionControl/index.jsx

```diff
@@ -60,7 +60,11 @@ const defaultProps = {
 const SortableListGroupItem = SortableElement(ListGroupItem);
 const SortableListGroup = SortableContainer(ListGroup);
 const SortableDragger = SortableHandle(() => (
-  <i className="fa fa-bars text-primary" style={{ cursor: 'ns-resize' }} />
+  <i
```

Comment:
Can't you assign `img` role here (if it's not already available)? Then use `aria-label` in `getByRole`?